src/Engine/UnitActionLifecycles/LoadLifecycle.java

```diff
@@ -168,6 +168,7 @@ public void performEvent(MapMaster gameMap)
         gameMap.removeUnit(unitCargo);
         unitCargo.x = -1;
         unitCargo.y = -1;
+        unitCargo.isTurnOver = true;
```

Comment:
This is why I probably did the adjacent-units thing before...

Reply:
haks r badd

Reply:
Hacks are bad, yes.
I suppose the *correct* fix would be to make MoveEvent unconditionally do the turn-over, fuel-burn, and fog-reveal?
At that point you'd get everything but the actual change in position, which is obviated by the Join or Load that will be right after it anyhow.